start-from-zero/index.html

```diff
@@ -4,36 +4,41 @@
     <meta charset="UTF-8">
     <title>index</title>
     <link rel="stylesheet" href="https://necolas.github.io/normalize.css/7.0.0/normalize.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet">
     <link rel="stylesheet" href="index.css">
 </head>
 <body>
     <section class="feedback">
         <div class="container">
-            <div class="column-30">
-                <div class="blue-line">Interview Feedback</div>
-                <form action="" class="user-input">
-                    <div class="field">
-                        <input id='input-zero' type="text" name='username' placeholder="Username">
-                    </div>
-                    <div class="field">
-                        <input id='input-zero' type="password" name='password' placeholder="Password">
-                    </div>
-                    <button class="button" type='submit'>Login</button>
-                </form>
+            <div class="row">
+               <div class="col-md-3 form-column">
+                    <div class="blue-line">Interview Feedback</div>
+                    <form action="" class="user-input">
+                        <div class="field">
+                            <input id='input-zero' type="text" name='username' placeholder="Username">
```

Comment:
So, you have at least two `input` elements with the same ID. What does that tell us? :)